azavea/echo-locator
# Make anonymous browsing site-configurable

## Overview

Add `env.yml` variable for whether anonymous browsing added in #68 should be enabled or not.


### Demo

![image](https://user-images.githubusercontent.com/960264/57162097-3f4f8a00-6dbb-11e9-931f-9757cb5128dc.png)


### Notes

The purpose of this is to temporarily disable anonymous site browsing until official launch.


## Testing Instructions

 * Add `ALLOW_ANONYMOUS: false` to `taui/configurations/default/env.yml`, or leave it undefined
 * `cd taui && yarn start`
 * Log out or clear session
 * Login page should have no anonymous browsing option
 * Should not be able to navigate directly to `/map`, `/search`, or `/profile` pages
 * Set `ALLOW_ANONYMOUS` to true and restart development server
 * Should be able to browse as anonymous user

Closes #172.
